core/storage/keyvalue.hpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_KEYVALUE_HPP
+#define KAGOME_KEYVALUE_HPP
+
+#include <outcome/outcome.hpp>
+#include "common/buffer.hpp"
+
+namespace kagome::storage {
+
+  class KeyValue {
+   public:
+    /**
+     * @brief Store value by key
+     * @param key non-empty byte buffer
+     * @param value byte buffer
```

Comment:
Please add description of.return value

Reply:
return value is arbitrary byte buffer, what can I write there?